product_docs/docs/bdr/4/conflicts.mdx

```diff
@@ -5,175 +5,172 @@ title: Conflicts
 ---
 
 BDR is an active/active or multi-master DBMS. If used asynchronously,
-writes to the same or related row(s) from multiple different nodes can
+writes to the same or related rows from multiple different nodes can
 result in data conflicts when using standard data types.
 
-Conflicts aren't ERRORs - they are events that can be detected and resolved
-automatically as they occur by BDR, in most cases. Resolution depends upon the
-nature of the application and the meaning of the data, so it is important that
+Conflicts aren't errors. In most cases, they are events that BDR can detect and resolve as they occur. 
+Resolution depends on the
+nature of the application and the meaning of the data, so it's important that
 BDR provides the application a range of choices as to how to resolve conflicts.
 
-By default, conflicts are resolved at row level. That is, when changes from two
-nodes conflict, we pick either the local or remote tuple and discard the other
-one. For example, we may compare commit timestamps for the two conflicting
-changes, and keep the newer one. This ensures that all nodes converge to the
-same result, and establishes commit-order-like semantics on the whole cluster.
+By default, conflicts are resolved at row level. When changes from two
+nodes conflict, BDR picks either the local or remote tuple and discards the other
```

Comment:
As Kelly mentioned on slack, I don't think that adding "BDR" is a positive change here, given the BDR, PG Distributed duality. Maybe the passive tone of  "either the local or remote tuple is picked and the other
Is discarded" would be better in these kinds of sentences.

Reply:
reverted to Kelly's rewrite (with just a tiny bit of editing)